Helper/ConfigHelper.php

```diff
@@ -271,6 +272,11 @@ public function useSecureUrlsInFrontend($storeId = null)
         );
     }
 
+    public function getBackendSearchEngine($storeId = null)
+    {
+        return $this->configInterface->getValue(self::BACKEND_SEARCH_ENGINE, ScopeInterface::SCOPE_STORE, $storeId);
+    }
+
```

Comment:
We can pull this information from \Magento\Search\Model\EngineResolver::getCurrentSearchEngine in Common.php instead of getting the information here.